### ISSUE TITLE
Method to read from path like routes to normal path

### ISSUE THREAD

**Issue by alexsad** [Belphemur/node-json-db#686]
I want to do this:
```js
const routerPathStyle = '/recipes/65464646155/nested/88488'
```
and get this:

```js
const routerPathStyle = '/recipes/65464646155/nested/88488'
  const normalPath = await db.toPath(routerPathStyle)
/// return '/recipes[1]/nested[1]'
```
with this data

```
            const recipe_1 = {
                id: '78687873783', 
                name: 'Gratin', 
                category: 'Dish',
            }
            const recipe_2 = {
                id: '65464646155',
                name: 'Cheesecake',
                category: 'Dessert',
                nested: [
                    {
                        id: '458445',
                        name: 'test-1',
                    },
                    {
                        id: '88488',
                        name: 'test-2',
                    },
                    {
                        id: '458455',
                        name: 'test-3',
                    },
                ],
            }
            const recipe_3 = {
                id: '12335373873',
                name: 'Soupe',
                category: 'Starter',
            }
            await db.push('/recipes[0]', recipe_1, true)
            await db.push('/recipes[1]', recipe_2, true)
            await db.push('/recipes[2]', recipe_3, true)
```

**Comment by alexsad** [comment]
pr
https://github.com/Belphemur/node-json-db/pull/687

**Comment by alexsad** [comment]
The main idea is to simplify access to an object's path using the route style by always searching for the "id" property of objects within a list.
eg.

```js
{
objects: [ 
{
   id:'4xs4s4',  
    list: [ 
      {id: '1' , desc: 'ssss', itens:[]},
      {id: '2' , desc: 'ssss', itens: []},
      {id: '3' , desc: 'ssss', itens:[
        {id:'21x6', name:'sss',
      ] },
    ]  
 }
 },
 { .... },
}
```

can be access by eg.

```js
const path= await db.toPath('/objects/4xs4s4/list/3/itens/21x6')
db.delete(path)
```

I don't know if I knew how to explain it well, but every time I need to get the path of an item more deeply I have to use getIndex more and more.
So with this method I can simplify searches for deletion and update.
Just an observation, I'm terrible at naming methods and variables, so feel free to modify if you're going to accept the PR.

**Comment by Belphemur** [comment]
Fixed by #688
